nixos/modules/installer/tools/tools.nix

```
@@ -120,7 +120,11 @@ in
         # Some programs need SUID wrappers, can be configured further or are
         # started in user sessions.
         # programs.mtr.enable = true;
-        # programs.gnupg.agent = { enable = true; enableSSHSupport = true; };
+        # programs.gnupg.agent = {
+        #   enable = true;
+        #   enableSSHSupport = true;
+        #   flavour = "gtk2";
```

GTK 2 is not maintained any more, let’s use GTK 3.

```suggestion
        #   flavour = "gnome";
```

---

done.